### ISSUE TITLE
Unexpected environment preparation error: Error response from daemon: OCI runtime create failed

### ISSUE THREAD

**Issue by chiubaka** [chiubaka/circleci-orb#4]
Seeing this whenever I attempt to run the local `circleci execute` commands, which would allow me to test and run parts of my orb locally.

Here's the full error:
```
Error: 
Unexpected environment preparation error: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:385: applying cgroup configuration for process caused: cannot enter cgroupv2 "/sys/fs/cgroup/docker/6976edaa676d3681c80bf9f2e116a2d62bd1655e66477fc8a73cc1e4c7742cc4/7afcf63907680c0cc9b08f62a2745b8dac7aa1c66bc03664fb35b88452a8babe" with domain controllers -- it is in an invalid state: unknown
```

Looks like there's [a](https://github.com/CircleCI-Public/circleci-cli/issues/676) [few](https://github.com/CircleCI-Public/circleci-cli/issues/672) [threads](https://github.com/CircleCI-Public/circleci-cli/issues/589) about this out there in CircleCI's GitHub repos.

Basically, it sounds like a bug in Docker that appears on macOS in more recent versions of the CircleCI CLI. My options seem to be:
- Downgrade to Docker Desktop v4.2.0 (not excited about this)
- Attempt [this workaround(https://github.com/CircleCI-Public/circleci-cli/issues/676#issuecomment-1107030440)
